wallets/provider-solflare/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import type {
+  CanSwitchNetwork,
+  Connect,
+  WalletInfo,
+} from '@rango-dev/wallets-shared';
+import type { BlockchainMeta, SignerFactory } from 'rango-types';
+
+import { Networks, WalletTypes } from '@rango-dev/wallets-shared';
+import Solflare from '@solflare-wallet/sdk';
+import { solanaBlockchain } from 'rango-types';
+
+import signer from './signer';
+
+const WALLET = WalletTypes.SOLFLARE;
+
+export const config = {
+  type: WALLET,
+};
+
```

Comment:
It seems you didn't implement change account.

Reply:
Implemented it but it just works when switching to an account which you already have given the permission to connect to the app and you checked the "auto connect" option.